src/components/badgedLink/BadgedLink.js

```diff
@@ -83,6 +84,8 @@ BadgedLink.propTypes = {
   // disabled: PropTypes.bool,
   /** The icon displayed inside the button. */
   icon: PropTypes.element.isRequired,
+  /** The aligment of the icon inside the button. */
+  iconAligment: PropTypes.oneOf(['center', 'bottom']),
```

Comment:
I'm assuming there's a use case for this somewhere? Could you show us a use case that would need "bottom" icon alignment instead of centered?